src/ray/common/ray_config_def.h

```diff
@@ -79,6 +83,9 @@ RAY_CONFIG(int64_t, max_num_to_reconstruct, 10000)
 /// regular raylet fetch timeout handler.
 RAY_CONFIG(int64_t, raylet_fetch_request_size, 10000)
 
+/// The maximum number of active object IDs to report in a heartbeat.
+RAY_CONFIG(int64_t, raylet_active_object_ids_size, 1000)
```

Comment:
```suggestion
RAY_CONFIG(int64_t, raylet_max_active_object_ids, 1000)
```

Reply:
`max_active_object_ids` seems more obvious since it's number of object ids, not object sizes.